channel/src/main/java/com/synopsys/integration/alert/channel/jira/server/action/JiraServerGlobalTestAction.java

```diff
@@ -55,12 +57,12 @@ protected boolean isAppMissing(FieldUtility fieldUtility) throws IntegrationExce
     }
 
     @Override
-    protected boolean isUserMissing(FieldUtility fieldUtility) throws IntegrationException {
+    protected boolean canUserViewProjects(FieldUtility fieldUtility) throws IntegrationException {
         JiraServerProperties jiraProperties = jiraServerPropertiesFactory.createJiraProperties(fieldUtility);
         JiraServerServiceFactory jiraServerServiceFactory = jiraProperties.createJiraServicesServerFactory(logger, gson);
-        UserSearchService userSearchService = jiraServerServiceFactory.createUserSearchService();
-        String username = jiraProperties.getUsername();
-        return userSearchService.findUserByUsername(username).stream().map(UserDetailsResponseModel::getName).noneMatch(email -> email.equals(username));
+        ProjectService projectService = jiraServerServiceFactory.createProjectService();
+        List<ProjectComponent> projects = projectService.getProjects();
```

Comment:
This could cause out-of-memory issues if the user has access to tens of thousands of projects.